### PR TITLE
Javascript/javascript1/week3

### PR DESCRIPTION
**Pull request by soujanyapolavarapu**

### CODE REVIEW

File: javascript/javascript1/week3/CactusIO-interactive.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  }
+  else{
+    console.log('You have reached your limit, no more smartphoning for you!');
+  }
+}
+
+
+//Show my status
+function showStatus(){
+  if(activities.length == 0){
```
Comment (aina21) [inline javascript/javascript1/week3/CactusIO-interactive.js:27]: ===

File: javascript/javascript1/week3/Item-array-removal.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+
+const nameToRemove = "Ahmad";
+
+names.splice(1,1);
```
Comment (aina21) [inline javascript/javascript1/week3/Item-array-removal.js:14]: try to find value in the array with for loop

File: javascript/javascript1/week3/NOnoN0nOYes-Note-taking-app.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  notes.push({
+    content: content,
+    id: id
+  });
+}
+
+
+//Get a note
+function getNote(id) {
+  if (id == '' || typeof(id) == 'string') {
```
Comment (aina21) [inline javascript/javascript1/week3/NOnoN0nOYes-Note-taking-app.js:25]: if(typeof id !== 'number')

File: javascript/javascript1/week3/NOnoN0nOYes-Note-taking-app.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    console.log('Id neither blank nor string');
+  } else {
+    for (i = 0; i < notes.length; i++) {
+      if (id == notes[i].id) {
```
Comment (aina21) [inline javascript/javascript1/week3/NOnoN0nOYes-Note-taking-app.js:29]: ===

File: javascript/javascript1/week3/NOnoN0nOYes-Note-taking-app.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        var idFound = 'found';
```
Comment (aina21) [inline javascript/javascript1/week3/NOnoN0nOYes-Note-taking-app.js:30]: don't use var,
when you have a get function, you need to return the value.

File: javascript/javascript1/week3/Series-duration-of-my-life.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    minutes: 30,
+  },
+];
+var avgLifeSpan = 80;
+const hoursInDay = 24;
+const daysInYear = 365.5;
+//console.log('soujanya');
+console.log(seriesDurations);
+//console.log(seriesDurations[0].title);
+var durationOfGameOfThronesInHours= (seriesDurations[0].days * hoursInDay ) + seriesDurations[0].hours ;
```
Comment (aina21) [inline javascript/javascript1/week3/Series-duration-of-my-life.js:27]: avoid var

File: javascript/javascript1/week3/Series-duration-of-my-life.js [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+//console.log(seriesDurations[1].title);
+var durationOfJustAddMagicInHours= (seriesDurations[1].days * hoursInDay ) + seriesDurations[1].hours + (seriesDurations[1].minutes/60) ;
+//console.log(durationOfJustAddMagicInHours);
+//console.log(seriesDurations[2].title);
+var durationOfMystoryCityInHours= (seriesDurations[2].days * hoursInDay ) + seriesDurations[2].hours + (seriesDurations[2].minutes/60) ;
+//console.log(durationOfMystoryCityInHours);
+logOutSeriesText();
+
+function logOutSeriesText() {
+var avgLifeSpanInHours = daysInYear * hoursInDay * avgLifeSpan;
```
Comment (aina21) [inline javascript/javascript1/week3/Series-duration-of-my-life.js:38]: same

**Review by aina21 — CHANGES_REQUESTED** [review]
good job 👍

File: javascript/javascript1/week3/CactusIO-interactive.js (hunk 1 above)
Comment (soujanyapolavarapu) [reply javascript/javascript1/week3/CactusIO-interactive.js:27]: Ok I will update.Thankyou

File: javascript/javascript1/week3/NOnoN0nOYes-Note-taking-app.js (hunk 4 above)
Comment (soujanyapolavarapu) [reply javascript/javascript1/week3/NOnoN0nOYes-Note-taking-app.js:29]: ok

File: javascript/javascript1/week3/NOnoN0nOYes-Note-taking-app.js (hunk 3 above)
Comment (soujanyapolavarapu) [reply javascript/javascript1/week3/NOnoN0nOYes-Note-taking-app.js:25]: ok thankyou

File: javascript/javascript1/week3/NOnoN0nOYes-Note-taking-app.js (hunk 5 above)
Comment (soujanyapolavarapu) [reply javascript/javascript1/week3/NOnoN0nOYes-Note-taking-app.js:30]: ok...will update

File: javascript/javascript1/week3/Series-duration-of-my-life.js (hunk 6 above)
Comment (soujanyapolavarapu) [reply javascript/javascript1/week3/Series-duration-of-my-life.js:27]: sure

File: javascript/javascript1/week3/Series-duration-of-my-life.js (hunk 7 above)
Comment (soujanyapolavarapu) [reply javascript/javascript1/week3/Series-duration-of-my-life.js:38]: ok.Thankyou for your comments, will update soon

File: javascript/javascript1/week3/Item-array-removal.js (hunk 2 above)
Comment (soujanyapolavarapu) [reply javascript/javascript1/week3/Item-array-removal.js:14]: //Thankyou @Ida i have updated.
const nameToRemove = "Ahmad";
//names.splice(1,1);
for (i=0; i<names.length; i++){
  if(names[i] === nameToRemove ){
    names.splice(i,1);
    break;
  }

}

**Pull request merged**